### ISSUE TITLE
Desfase climas

### ISSUE THREAD

**Issue by ecoeficiente** [pachi/eplusctekit#2]
Buenas, creo que ya  lo hemos comentado alguna vez pero los ficheros de clima están desfasados (1 hora al menos). ¿Cómo está hecha la conversión?
Los de Canarias cuadran algo mejor, por eso del GMT+0
Omitir la ecuación del tiempo supone +-15 minutos (podríamos cuestionarnos obvierlo), pero al menos el desfase respecto al GMT sí debería incluirse...

**Comment by pachi** [comment]
Los archivos E+ deben estar en hora local, mientras que los archivos MET puedes comprobar, viendo la altura solar, que están en hora solar. Además, los MET indican que la hora asignada a la 1 es lo que ocurre en la media hora anterior (o posterior, nunca me acuerdo), de modo que al traducir esos puntos a horas E+ la cosa todavía se complica más.
Debería publicar los script de conversión en una carpeta adicional. No lo he hecho todavía porque no es sencillo instalar las dependencias para hacer funcionar el programa. En cuanto saque tiempo lo miro.

**Comment by pachi** [comment]
Piensa que estamos en la misma longitud que Reino Unido, así que ya hay un desfase de partida de una hora entre hora local y solar, además de la desviación debida a la posición de la tierra que da la ecuación del tiempo.

**Comment by ecoeficiente** [comment]
Sí, a eso me refería, ya sé que los met están en solar y que hay que pararlos a local. Pero me parece que no están bien pasados... El convenio de E+ es el valor medio de la hora anterior, que parece ser el mismo que los met. Digo parece porque no está escrito en ninguna parte que yo sepa y tan sólo lo deduzco por la simetría de los valores de altura y azimuth...
Es bastante jaleo aclararse con las diferencias horarias... podían haberlos publicado en hora local, en fin.

**Comment by ecoeficiente** [comment]
Por cierto, acabo de ver que han modificado algunos ficheros en la página oficial. Además de modificar la longitud de referencia para los canarios (que no lo pone en la explicación), añaden un Leeme.txt donde pone que son en hora solar. Curioso que los ficheros sean del 19 de abril de 2014 y se hayan publicado este mes:

> 19/01/2016
> ----------
> 
> - Archivos climáticos del CTE en formato .MET: Se han corregido los archivos
>   climáticos de las zonas A1, B1, alfa2 y alfa3 correspondientes a las Islas
>   Canarias, de modo que coincidan con los usados por la Herramienta Unificada
>   LIDER-CALENER.

**Comment by pachi** [comment]
Sí, esos archivos corresponden a lo realmente usado en HULC. Registré el tema del desfase, aunque intervienen varias cosas, incluida la interpelación final.
